addons/medical/functions/fnc_displayPatientInformation.sqf

```diff
@@ -158,30 +158,35 @@ if (_show == 1) then {
         // Handle the body image coloring
         private _availableSelections = [50, 51, 52, 53, 54, 55];
         {
-            private _red = 1;
-            private _green = 1;
-            private _blue = 1;
+            private _colorSelection = [1, 1, 1]; // RGB
 
             private _torniquet = _selectionTourniquet select _forEachIndex;
             if (_torniquet > 0) then {
-                _red = 0;
-                _green = 0;
-                _blue = 0.8;
+                _colorSelection = [0, 0, 0.8];
             } else {
                 private _bloodLoss = _selectionBloodLoss select _forEachIndex;
-                if (_bloodLoss > 0) then {
-                    _green = 0 max (0.8 - _bloodLoss);
-                    _blue = _green;
-                } else {
-                    private _damage = _selectionDamage select _forEachIndex;
-                    if (_damage > 0.1) then {
-                        _blue = 0 max (1 - _damage);
-                        _green = 0.5 max (1 - _damage / 2);
-                    };
-                };
+
+                private _frBL = 0 max (_bloodLoss / BLOOD_LOSS_RED_THRESHOLD) min 1;
+                private _colorInt = ceil ((_frBL * (BLOOD_LOSS_TOTAL_COLORS - 1)));
+
+                _colorSelection = [
```

Comment:
Should this array of colors be a macro?

Reply:
I'm going to tidy a bunch of stuff up once the functionality is finalised